src/test_l1_dsDisplay.c

```diff
@@ -502,8 +503,22 @@ void test_l1_dsDisplay_positive_dsGetEDID(void) {
             UT_ASSERT_EQUAL(result, dsERR_NONE);
 
             // Step 04: Call dsGetEDID() again with the same handle
-            result = dsGetEDID(displayHandle, edid2);
-            UT_ASSERT_EQUAL(result, dsERR_NONE);
+	    edid2->productCode = ut_kvp_getUInt32Field(ut_kvp_profile_getInstance(), "dsDisplay/EDID_Data/productCode");
```

Comment:
`ut_kvp_getUInt32Field(ut_kvp_profile_getInstance(), ` -> search & replace it with `UT_ASSERT_KVP_EQUAL_PROFILE_UINT32(` so you get `UT_ASSERT_KVP_EQUAL_PROFILE_UINT32( "dsDisplay/EDID_Data/productCode" )` - which makes it easier to read.

Reply:
done